server/player/player.go

```diff
@@ -1191,10 +1197,53 @@ func (p *Player) UseItem() {
 // the item started being used.
 func (p *Player) ReleaseItem() {
```

Comment:
shouldn't this also be applicable for the item.Releasable interface too? (talking about the documentation)

Reply:
Changed.